### PR TITLE
Increase Links per Page

### PR DESCRIPTION
**Pull request by itd-cba**
This is the requested change

<sup> Created from JetBrains using [CodeStream](https://codestream.com/?utm_source=cs&utm_medium=pr&utm_campaign=github*com)</sup>

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -1,2 +1,2 @@
 export const AUTH_TOKEN = 'auth_token';
-export const LINKS_PER_PAGE = 5;
\ No newline at end of file
+export const LINKS_PER_PAGE = 10;
```
Comment (itd-cba) [inline src/constants.js:2]: This is the change you wanted or ?

File: src/index.js [hunk 2]
```diff
@@ -68,4 +68,6 @@ ReactDOM.render(
         </ApolloProvider>
     </BrowserRouter>,
     document.getElementById('root')
-);
\ No newline at end of file
+);
+
+console.log("This is an debug comment")
```
Comment (itd-cba) [inline src/index.js:73]: This should be deleted

File: src/components/Search.js [hunk 3]
```diff
@@ -55,4 +55,8 @@ const Search = () => {
     );
 };
 
-export default Search;
\ No newline at end of file
+export default Search;
+
+console.log('Please give me feedback ')
```
Comment (benni-stein) [inline src/components/Search.js:60]: This log is always logged as soon as the file loads.
Should it be wrapped in a function or is logged on purpose?

File: src/index.js (hunk 2 above)
Comment (benni-stein) [inline src/index.js:73]: this is always logged as well, ok?

**Pull request closed without merging**